src/components/Admin/index.jsx

```diff
@@ -211,14 +270,23 @@ class Admin extends React.Component {
   }
 
   renderDownloadButton() {
-    const { actionSlug } = this.props;
+    const { actionSlug, intl } = this.props;
     const tableMetadata = this.getMetadataForAction(actionSlug);
+    let downloadButtonLabel;
+    if (actionSlug) {
+      downloadButtonLabel = intl.formatMessage({
+        id: 'admin.portal.lpr.current.report.csv.download',
+        defaultMessage: 'Download current report (CSV)',
```

Comment:
where we are handling `Download full report (CSV)` ?

Reply:
`Download full report (CSV)` is handled [here](https://github.com/openedx/frontend-app-admin-portal/blob/3fcadf5f1337b2f1cacebb1740735e9dff5b0498/src/components/DownloadCsvButton/index.jsx#L40)